CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 ## Unreleased:
 
+### API
+#### Enhancements
+- Parsing of the W3C Baggage header has been optimized.
+
+### SDK
```

Comment:
Think we need to callout the small behavior change in https://github.com/open-telemetry/opentelemetry-java/pull/3244

Reply:
i'll add it. good call.